### PR TITLE
[LCS-353] - Update /agent-details 

### PR DESCRIPTION
**Pull request by sulthan-ahmed**
also company field is optional

https://jira.digital.homeoffice.gov.uk/browse/LCS-353

screen shot 2017-11-14 at 10 17 14

### CODE REVIEW

File: apps/right-to-rent-check/translations/src/en/fields.json [hunk 1]
```diff
@@ -117,7 +117,6 @@
   },
   "agent-email-address": {
     "label": "Email address",
-    "hint": "We will use this to let you know the outcome of the check",
```
Comment (JoeChapman) [inline apps/right-to-rent-check/translations/src/en/fields.json]: does this relate to a ticket?

Comment (lennym) [reply apps/right-to-rent-check/translations/src/en/fields.json]: I'd have assume LCS-353, but then the content changes here don't match what's in the ticket. @sulthan-ahmed?

Comment (sulthan-ahmed) [reply apps/right-to-rent-check/translations/src/en/fields.json]: yep that's the ticket https://jira.digital.homeoffice.gov.uk/browse/LCS-353

Most of the content is actually there, I just had to remove this hint and make the company field optional.  I've now updated my PR & commit with the ticket number.  I've also included a screenshot.  I hope that helps?  I can ask @Nadiahuq to review as it now has the screenshot

Comment (Nadiahuq) [reply apps/right-to-rent-check/translations/src/en/fields.json]: Reviewed and happy. Thanks @sulthan-ahmed

Comment (sulthan-ahmed) [reply apps/right-to-rent-check/translations/src/en/fields.json]: Thanks @Nadiahuq

**Pull request merged**